docs/jwst/source_catalog/main.rst

```diff
@@ -53,12 +53,6 @@ semimajor and semiminor axis lengths, orientation of the major axis,
 and sky coordinates at corners of the minimal bounding box enclosing
 the source.
 
-.. Note::
```

Comment:
I agree that this note can now be removed, given the updates to the code, but is there anywhere else in the docs where we at least mention that all of the photometric errors listed above are computed based on the ERR (model.error) array of the input data, as well as mentioning that the ERR array contains all noise/error sources (e.g. Poisson, read noise, flat-field, ...) computed upstream in the pipeline? If not, I think it would be good to add just a sentence or two about this somewhere.

Reply:
I added a note in the docs about the source of the photometric errors.  I stated that this "total" error array *includes* source Poisson noise.  "total" here means combining all of the propagated error planes, but I'm don't think it includes the flat-field (@jdavies-st ?).  In any case, "total" could change in the future if more error arrays are propagated.  I think the important thing to point out here is that the source Poisson noise is included, which may be unexpected for those familiar with HST data.

Reply:
I think our ERR does in fact also include flat-field variance, but you're right that it's probably not necessary to list every kind of noise source that is included. As you suggest, the important one is simply the Poisson term, which is different than HST. So wording like "... total error array, which includes Possion noise, ..." should be fine.